qutebrowser/misc/utilcmds.py

```diff
@@ -234,3 +234,17 @@ def repeat_command(win_id, count=None):
     cmd = runners.last_command[mode_manager.mode]
     commandrunner = runners.CommandRunner(win_id)
     commandrunner.run(cmd[0], count if count is not None else cmd[1])
+
+@cmdutils.register(debug=True,name='debug-log-capacity')
+def log_capacity(capacity: int):
+    """Choose number of lines for your log.
```

Comment:
I think this might be confusing for users who don't know what this does exactly - maybe something like *"Change the number of log lines to be stored in RAM."* would be better?